pgsmo/utils/querying.py

```diff
@@ -1,70 +0,0 @@
-# --------------------------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for license information.
-# --------------------------------------------------------------------------------------------
-
-from typing import List, Mapping, Tuple
-
-from psycopg2.extensions import Column, connection, cursor      # noqa
-
-
-class ServerConnection:
-    """Wrapper for a psycopg2 connection that makes various properties easier to access"""
-
-    def __init__(self, conn: connection):
-        """
-        Creates a new connection wrapper. Parses version string
```

Comment:
pgsmo is something that would be specific to Postgres. I believe we will be adding MySQL smo to handle mysql\mariadb objects. Lets think whether we have to generalize the smo part.

Reply:
Sure thing! I have some ideas for this that I can share with you soon.